### ISSUE TITLE
Request for support of paragraph-by-paragraph translation functionality.

### ISSUE THREAD

**Issue by shang-zhen** [FilipePS/Traduzir-paginas-web#727]
The current method divides the translation based on tags. I have noticed that some inline tags, such as "span strong i," are being split during translation, resulting in inaccurate translations of entire sentences.

In this context class in programming is translated into school class:
> Suppose we have an <code>AuthRepository</code> class we use as a wrapper for the <code>FirebaseAuth</code> class:
假设我们有一个 AuthRepository我们用作包装器的类 FirebaseAuth班级

> paragraph-by-paragraph:
假设我们有一个 AuthRepository 类，用作 FirebaseAuth 类的包装器

The quality of the second translation is significantly better than the first.

If the translation were done based on paragraphs, the effect would be much better.
Could you display the translated content below each paragraph? It would be helpful to remove any inline tags from the original paragraph, extract the text only, translate it, and place it below the paragraph.

**Comment by 0x391F** [comment]
Yeah, https://addons.mozilla.org/zh-CN/android/addon/immersive-translate/ has this feature, it's very, very convenient!

**Comment by 0x391F** [comment]
Yeah, https://addons.mozilla.org/zh-CN/android/addon/immersive-translate/ has this feature, it's very, very convenient!
